Number_Coding.py

```diff
@@ -0,0 +1,17 @@
+MAJOR_COLORS_LIST = ['White', 'Red', 'Black', 'Yellow', 'Violet']
```

Comment:
redefinition of colors. this needs to be avoided.

Reply:
I have updated as per the feedback